Source/WebCore/platform/graphics/cg/UTIRegistry.cpp

```diff
@@ -50,7 +50,6 @@ const MemoryCompactLookupOnlyRobinHoodHashSet<String>& defaultSupportedImageType
             "public.png"_s,
             "public.tiff"_s,
 #if !PLATFORM(WIN)
-            "public.jpeg-2000"_s,
```

Comment:
This is enough to stop Safari from displaying JPEG 2000 images? If so, great. The Source/WebCore/platform/image-decoders/jpeg2000/ only needs to exist so long as Safari supports it.